scimath/StatsFramework/ClassicalQuantileComputer.tcc

```diff
@@ -753,7 +753,7 @@ ClassicalQuantileComputer<CASA_STATP>::_dataFromMultipleBins(
         auto iIdx = idxSet.cbegin();
         auto eIdx = idxSet.cend();
         const auto& maxBinLims = iDesc->getMaxBinLimits();
-        if (iSameVal->null()) {
+        if (! *iSameVal) {
```

Comment:
I don't think the dereference operator should be there, should it? Shouldn't it be
`if (! iSameVal) {`
?